Bridges/BridgeInterface.php

```diff
@@ -29,8 +29,8 @@ public function getStaticDirectory();
     /**
      * Handle a request using a HttpKernelInterface implementing application.
      *
-     * @param \React\Http\Request $request
-     * @param \PHPPM\React\HttpResponse $response
+     * @param RequestInterface $request
+     * @return ResponseInterface
      */
-    public function onRequest(\React\Http\Request $request, HttpResponse $response);
+    public function onRequest(RequestInterface $request);
```

Comment:
Since we're doing a BC break anyway I'm wondering if we shouldn't implement https://github.com/php-fig/fig-standards/blob/master/proposed/http-middleware/middleware-meta.md right away? On the other side that could as well be a topic for a 2.0 release.